docs/gambit/mutation-verifier.md

```diff
@@ -49,10 +49,13 @@ Here is an example configuration file:
 {
     "project_folder" : "Test/10Power",
     "run_script" : "Test/10Power/runDefault.sh",
-    "gambit" : {
-        "filename" : "Test/10Power/TenPower.sol",
-        "solc" : "solc8.10"
-    },
+    "gambit" : 
```

Comment:
@teryanarmen btw `{}` should also work. Did it not?

Reply:
I think I made this PR too quickly without fully troubleshooting the issue. I will remove.